### DISCUSSION TITLE
[Question] Anymal USD/URDF Parameters

### DISCUSSION THREAD

**Discussion by vassil-atn** [isaac-sim/IsaacLab#1113] [Q&A]
### Question
Hi,

I've been playing around with the Anymal-C env and had a few questions regarding it.

1. Should the velocity limit in the USD (imported from the URDF) match the real velocity limit, i.e. 7.5 rad/s for the anymal-C ? Currently the limit is 20 in the URDF used for the anymal-c usd (https://github.com/isaac-orbit/anymal_c_simple_description/blob/master/urdf/anymal.urdf). 

2. The same URDF has the joint position limits removed. Is there a particular reason for this?

3. Why is enable_self_collisions set to True for the Anymal envs?

For Anymal-D 1) and 2) seem to be set to the true values, but that doesn't seem to be the case for Anymal-C. Is there an issue with how the simulator handles those limits that led to this decision?

**Comment by StoneT2000** [comment]
I recently was trying to get a sense of how Isaac was improving quadruped simulation speeds and can guess at some of these

1. No idea
2. I am not sure why joint limits are removed but somehow the USD file they use for simulation has small joint limits where it makes it difficult to easily flip over the robot when doing RL
3. It probably does not matter. Isaac uses a simplified collision mesh where in fact each collision mesh is quite far apart from the others, and many collision meshes are removed to improve simulation speed

**Comment by Dhoeller19** [comment]
Hi @vassil-atn,
these changes have rather been arbitrary. You can set back the limit to 7.5 rad/s. Enabling self collisions is often important to prepare the policy for sim-to-real transfer.
